examples/pre-push/prevent-force-push-to-master

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+protected_branch='master'
+
+policy='[Policy] Never force push or delete the '$protected_branch' branch! (Prevented with pre-push hook.)'
+
+current_branch=$(git symbolic-ref HEAD | sed -e 's,.*/\(.*\),\1,')
+
+parent_pid=$(ps -oppid= -p $PPID)
+push_command=$(ps -ocommand= -p $parent_pid)
```

Comment:
the current process is a node.js one. So to get the original git command we need to run ps with parent pid. It looks trivial or maybe I didn't understand the question